GenderPayGap.Core/Models/DownloadResult.cs

```diff
@@ -6,6 +6,7 @@ public class DownloadResult
     {
 
         public string EmployerName { get; set; }
+        public long OrganisationId { get; set; }

```

Comment:
Recently all the user facing wording has been updated - "organisation" being changed to "employer". Therefore, I think we should rename this field

Reply:
Sure, will do that! Thanks 🙏.